formdata-methods/index.html

```diff
@@ -0,0 +1,32 @@
+---
+feature_name: FormData methods for inspection and modification
+chrome_version: 50
+feature_id: 5701572055007232
+---
+
+<h3>Background</h3>
+<p>The <code>FormData</code> object helps to programatically send encoded form data to a server with an XHR. You can even generate it from a real <code>&lt;form&gt;</code>.</p>
+<p>These methods allow inspection and modification of the content of the <code>FormData</code>. Previously, this object could only be appended to, and never inspected. For a full list of methods, <a href="https://developer.mozilla.org/en/docs/Web/API/FormData" target="_blank">check MDN</a>.</p>
+
+{% capture initial_output_content %}
+Submit the form to see its content from a <code>FormData</code>.
+{% endcapture %}
+{% include output_helper.html initial_output_content=initial_output_content %}
+
+<form style="border: 1px solid #777; padding: 20px; border-radius: 2px;" id="form">
+  <input name="username" name="username" type="text" value="alpha" placeholder="Username" /><br />
```

Comment:
General style throughout the sample repo has been to omit the `/` on [void elements](http://w3c.github.io/html/single-page.html#void-elements) like `<input>` and `<br>`.
Not a huge deal, but it would be nice to be consistent.